user/user.go

```diff
@@ -275,3 +275,87 @@ func (uh *UserHandler) GetUserOrganizations(response http.ResponseWriter, reques
 
 	utils.GetSuccess("user organizations retrieved successfully", orgs, response)
 }
+
+// Create a new user from UUID guest invite sent to user and a supplied password.
+func (uh *UserHandler) CreateUserFromUUID(w http.ResponseWriter, r *http.Request) {
+	w.Header().Add("content-type", "application/json")
+
+	user_collection, orgInvite := "users", "organizations_invites"
```

Comment:
Lets also move these constants to the model file. You can do this correction other relevant places in the user file. I started this in organization too.

Reply:
I will make the necessary changes